index.bs

```diff
@@ -0,0 +1,24 @@
+<pre class='metadata'>
+Title: Isolated Contexts
+Shortname: isolated-contexts
+Level: 1
+Group: wicg
+Status: w3c/CG-DRAFT
+ED: https://wicg.github.io/isolated-web-apps
```

Comment:
If we're naming this specification "Isolated Contexts" then let's give it the URL `https://wicg.github.io/isolated-web-apps/isolated-contexts` (name it `isolated-contexts.bs`) so that we have the option of creating more specifications under this repository later without causing ourselves issues.

Reply:
Good point, done!